electron.d.ts error using TS2.6.0-rc in strict mode

```
node_modules/electron/electron.d.ts(5390,13): error TS2430: Interface 'WebviewTag' incorrectly extends interface 'HTMLElement'.
  Types of property 'addEventListener' are incompatible.
      Type '{ (event: "load-commit", listener: (event: LoadCommitEvent) => void, useCapture?: boolean | undef...' is not assignable to type '{ <K extends "error" | "waiting" | "progress" | "ended" | "change" | "input" | "select" | "abort"...'.
        Types of parameters 'listener' and 'listener' are incompatible.
          Type 'EventListenerOrEventListenerObject' is not assignable to type '(event: LoadCommitEvent) => void'.
            Type 'EventListenerObject' is not assignable to type '(event: LoadCommitEvent) => void'.
              Type 'EventListenerObject' provides no match for the signature '(event: LoadCommitEvent): void'.
```

I think it is due to this new strict change:
https://github.com/Microsoft/TypeScript/wiki/What's-new-in-TypeScript#strict-function-types

--- a/lib/module-declaration.js
+++ b/lib/module-declaration.js
@@ -105,6 +105,12 @@ const generateModuleDeclaration = (module, index, API) => {
         moduleAPI.push(`${method}(event: '${domEvent.name}', listener: (event: ${eventType}) => void${method === 'addEventListener' ? ', useCapture?: boolean' : ''}): this;`)
       }
     })
+
+    // original overloads copied from HTMLElement, because they are not inherited
+    moduleAPI.push(`addEventListener<K extends keyof HTMLElementEventMap>(type: K, listener: (this: HTMLElement, ev: HTMLElementEventMap[K]) => any, useCapture?: boolean): void;`)
+    moduleAPI.push(`addEventListener(type: string, listener: EventListenerOrEventListenerObject, useCapture?: boolean): void;`)
+    moduleAPI.push(`removeEventListener<K extends keyof HTMLElementEventMap>(type: K, listener: (this: HTMLElement, ev: HTMLElementEventMap[K]) => any, useCapture?: boolean): void;`)
+    moduleAPI.push(`removeEventListener(type: string, listener: EventListenerOrEventListenerObject, useCapture?: boolean): void;`)
   }
 
   const returnsThis = (moduleMethod) => ['on', 'once', 'removeAllListeners', 'removeListener'].includes(moduleMethod.name)

